src/docs/index.md

```diff
@@ -37,6 +37,7 @@ V8 enables any C++ application to expose its own objects and functions to JavaSc
     - [Checklist for staging and shipping of WebAssembly features](/docs/wasm-shipping-checklist)
     - [Flake bisect](/docs/flake-bisect)
     - [Handling of ports](/docs/ports)
+    - [Official support](/docs/official-support)
```

Comment:
Please maintain alpha-sorting

Reply:
Hmm, this list doesn't seem alpha sorted at all. Maybe only the 3-4 items around it here. Where should I move it?

I thought from a logical perspective it should stay side-by-side with the port handling. It's the closest in meaning.

Reply:
Ok, nevermind then